addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -86,6 +86,14 @@ const LinkTools = Link.extend({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     */
+    _onAnyChange: function (e) {
+        if (!e.target.closest('input[type="text"]:not(#o_link_dialog_url_input)')) {
```

Comment:
Isn't there another way to target the correct input in the Link widget instead of redefining the function here ?